webpack.prod.js

```diff
@@ -0,0 +1,12 @@
+var webpack = require('webpack');
```

Comment:
is having two files how webpack deals with env?

Reply:
It's not the only way, but it's the way I chose. It let's us extract common config and then extend it per environment. https://simonsmith.io/organising-webpack-config-environments/